documentation/book/assembly-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,32 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-maintenance-time-windows-{context}']
+
+= Maintenance time windows for rolling updates
+
+Maintenance time windows allow you to schedule rolling updates of your Kafka and Zookeeper clusters to start at a convenient time.
+During a rolling update, pod instances in a cluster are updated incrementally, with no downtime.
+
+You configure a maintenance time window by entering an array of strings containing one or more {CronExpressions} in the `Kafka.spec.maintenanceTimeWindows` property.
+The cron expressions must be expressed in UTC.
```

Comment:
Cron expressions don't have any syntax for expressing a timezone, so the way this is worded isn't as clear (imho) as saying that "The cron expressions are interpreted as being in UTC".

Reply:
Good feedback; happy to change this.